packages/google_maps_flutter/google_maps_flutter_android/pigeons/messages.dart

```diff
@@ -41,6 +41,17 @@ class PlatformCircle {
   final Object json;
 }
 
+/// Pigeon equivalent of the Heatmap class.
+class PlatformHeatmap {
+  PlatformHeatmap(this.json);
+
+  /// The heatmap data, as JSON. This should only be set from
+  /// Heatmap.toJson, and the native code must interpret it according to the
```

Comment:
Can you link to the native code that is required to match the dart code here?

Reply:
This whole file needs work. I updated this to include the TODO that the other classes now have.